lib/md045.js

```diff
@@ -2,15 +2,19 @@
 
 "use strict";
 
-const { addError } = require("../helpers");
-const { filterByTypes } = require("../helpers/micromark.cjs");
+const { addError, nextLinesRe } = require("../helpers");
+const { filterByTypes, getHtmlTagInfo } = require("../helpers/micromark.cjs");
+
+// Regular expression for identifying alt attribute
+const altRe = /\salt=/i;
```

Comment:
For comparison, I use this elsewhere for similar purpose: `const idRe = /\sid\s*=\s*['"]?([^'"\s>]+)/iu;`. I'd like to try that here.

Reply:
Actually, there should maybe be a getHtmlAttributeRe(name) function for this.

Reply:
I'll implement this separately as it's a little more involved.